hedera-node/src/main/java/com/hedera/services/store/models/TokenConversion.java

```diff
@@ -0,0 +1,200 @@
+package com.hedera.services.store.models;
+
+/*-
+ * ‌
+ * Hedera Services Node
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.hedera.services.state.merkle.MerkleToken;
+import com.hedera.services.state.merkle.MerkleUniqueToken;
+import com.hedera.services.state.submerkle.EntityId;
+import com.hedera.services.store.AccountStore;
+
+import javax.annotation.Nullable;
+
+/**
+ * Performs MerkleToken {@link MerkleToken}  - Token {@link Token} - MerkleToken {@link MerkleToken} and
+ * MerkleUniqueToken {@link MerkleUniqueToken} - UniqueToken {@link UniqueToken} - MerkleUniqueToken {@link MerkleUniqueToken}
+ * conversions by preserving all the fields as expected.
+ */
+public final class TokenConversion {
+	private TokenConversion() {
+		throw new UnsupportedOperationException("Utility class");
```

Comment:
Add this class to [UtilsConstructorTest](https://github.com/hashgraph/hedera-services/blob/master/hedera-node/src/test/java/com/hedera/services/utils/UtilsConstructorTest.java).